### PR TITLE
Revert "workaround for section header regression (#937)"

### PR DESCRIPTION
**Pull request by bdattoma**
This reverts the workaround applied for the broken` Jump to section` menu. 

There is one piece not working without the workaround, i.e., adding multiple environments in the workbench, but it doesn't seem a product-related issue. Probably we need to change some of the html we use or increase the window size of the browser (cc: @lugi0 )

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource [hunk 1]
```diff
@@ -126,9 +120,9 @@ Add Environment Variables In Workbench
                     # The element we want to click might get pushed out of view on every loop, let's scroll by an approx
                     # amount of pixels for the block of elements that gets added.
                     IF    "${add_key_value_text}" == "Add another key / value pair"
-                        Execute Javascript    document.getElementsByClassName("pf-c-drawer__content")[1].scrollBy(0,200)  # robocop: disable
+                        Execute Javascript    document.getElementById("dashboard-page-main").scrollBy(0,200)  # robocop: disable
                     ELSE IF    "${add_key_value_text}" == "Add another variable"
-                        Execute Javascript    document.getElementsByClassName("pf-c-drawer__content")[1].scrollBy(0,700)  # robocop: disable
+                        Execute Javascript    document.getElementById("dashboard-page-main").scrollBy(0,700)  # robocop: disable
```
Comment (lugi0) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource:125]: These lines shouldn't be reverted

Comment (bdattoma) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource]: done! thanks

**Review by lugi0 — APPROVED** [review]

**Comment by bdattoma** [comment]
locally tested and it's working. Will test from jenkins tomorrow

**Comment by bdattoma** [comment]
entire suite tested on jenkins: `rhods-ci-pr-test/2015` 100% pass

**Review by jgarciao — APPROVED** [review]

**Pull request merged**